boards/arm/nrf52810_pca10040/board.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+/* Push button switch 0 */
+#define SW0_GPIO_PIN     13
```

Comment:
Can we do this via DTS instead since this is a new board.

Reply:
I would love to, but I think I would need some help here. Can you give me some pointers?

Reply:
You'll also need to pull in some changes from PR #7444 (meaning adding the dt nodes for GPIO etc).

Reply:
Very well. In that case I will wait until #7444 is merged, since this should not be merged before 1.12 is released anyway. Thanks for the pointers.